RELEASE_NOTES.rst

```diff
@@ -4,10 +4,10 @@ coala 0.8.1
 This bugfix release addressed the following issues:
 
 - The cache will be correctly invalidated when changing section targets now.
-  (https://github.com/coala-analyzer/coala/issues/2708)
+  (https://github.com/coala/coala/issues/2708)
```

Comment:
I wouldn't change stuff from release notes because they're written in the past. There is a redirect in place anyway, so there won't be broken links here anyway.